implementations.yaml

```diff
@@ -144,30 +168,22 @@ kems:
         alt_implementations:
           - clean
   hqc:
-    version: 0.0.4
-    notes: |
-      This implementation is not constant-time!
-      This means that it is not secure.
-
-      This crate may remove the ``leaktime`` implementation at any point.
+    version: 0.0.5
     schemes:
-      - name: hqc-128-1-cca2
-        implementation: leaktime
-      - name: hqc-192-1-cca2
-        implementation: leaktime
-      - name: hqc-192-2-cca2
-        implementation: leaktime
-      - name: hqc-256-1-cca2
-        implementation: leaktime
-      - name: hqc-256-2-cca2
-        implementation: leaktime
-      - name: hqc-256-3-cca2
-        implementation: leaktime
+      - name: hqc-rmrs-128
+        implementation: clean
+        avx2_implementation: avx2
+      - name: hqc-rmrs-192
+        implementation: clean
+        avx2_implementation: avx2
+      - name: hqc-rmrs-256
+        implementation: clean
+        # avx2_implementation: avx2
 
 
 signs:
   dilithium:
-    version: 0.3.3
+    version: 0.3.4
```

Comment:
This should be bumped to `0.4.0` as it's not compatible.

Reply:
Applied